### PR TITLE
[FEAT] PIN in payment description

### PR DESCRIPTION
**Pull request by supiiik**
Added PIN in payment description to make it easyer to find failed payment for example

### CODE REVIEW

File: lnurl.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ async def lnurl_params(
             "k1": p,
             "minWithdrawable": price_msat * 1000,
             "maxWithdrawable": price_msat * 1000,
-            "defaultDescription": device.title,
+            "defaultDescription": device.title + str(lnurldevicepayment.pin),
```
Comment (dni) [inline lnurl.py:196]:
```suggestion
            "defaultDescription": f"{device.title} - pin: {str(lnurldevicepayment.pin)}",
```

**Comment by supiiik** [comment]
Can you please remove string "- pin:"? This string can be easyly added in LNURLdevice title, for example "ATM - pin: ", but if that string is hard coded, than user have no choice to change it. For example I wanna use "ATM off" to make description in user wallet and also in lnbits payments list like "ATM off1234".

**Comment by dni** [comment]
wouldn't `ATM off - pin: 1234` be much nicer? we hardcoded the {pin} anyway. i can revert it if you want, but i think it is more descriptive for user who doesn't know what this number means. imagine `mydescription2` it it not obvious what 2 means if you don't know that the pin is appended.

**Comment by supiiik** [comment]
Yes, that's nicer, but uncustomizable. In my case I'm using it as an "ID" of payment, not "PIN", so it can be little bit confusing.
The best would be to have an option to append PIN after title or not. For example checkbox after the title in "new lnurldevice instance" and then the "- pin:" string is not needed, because user can choose, if he wanna show that PIN and if yes, than he can choose own text. Unfortunatelly, this is out of my Python skills :/

**Pull request merged**